src/tests/stress/pedsim/Ped_util/Regions.enc

```diff
@@ -332,19 +323,22 @@ active class Box
         case unused  => true
      end
   end
-  def move() : bool
+  def private yolo_item(a:Maybe[Item]) : Item
```

Comment:
this function is named `yolo_item` but what it's doing is extracting whatever is in an option type.
Doesn't it make more sense to write the following function:
```python
fun fromJust[a](x: Maybe[a]): a
    match x with
        case Just(value) => value
        case Nothing     => abort("Error: XXXX") -- or something
    end
end
```